### PR TITLE
Add Pokemonlevel to Trainerpage

### PR DESCRIPTION
**Pull request by pki0**
## Description
- Add Pokemonlevel to Trainer.page
- Know your own Pokemonlevel to see if it is worth to push 
- Source: https://docs.google.com/spreadsheets/d/19iql4aABmZ5oZ6YDE3LmZ8qcth3UoH52954WhjuiJow/edit#gid=771823706

## Screenshots (if appropriate):
image
image

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: core/js/trainer.content.js [hunk 1]
```diff
@@ -243,6 +243,93 @@ function printPokemon(pokemon,pokeimg_suffix,iv_numbers,locale){
...
+	else if (pokemon.cp_multiplier <= '0.767397' && pokemon.additional_cp_multiplier > '0') {pokemon_level = '36.5';}
+	else if (pokemon.cp_multiplier <= '0.773187' && pokemon.additional_cp_multiplier <= '0') {pokemon_level = '37';}
+	else if (pokemon.cp_multiplier <= '0.773187' && pokemon.additional_cp_multiplier > '0') {pokemon_level = '37.5';}
+	else if (pokemon.cp_multiplier <= '0.778933' && pokemon.additional_cp_multiplier <= '0') {pokemon_level = '38';}
+	else if (pokemon.cp_multiplier <= '0.778933' && pokemon.additional_cp_multiplier > '0') {pokemon_level = '38.5';}
+	else if (pokemon.cp_multiplier <= '0.784637' && pokemon.additional_cp_multiplier <= '0') {pokemon_level = '39';}
+	else if (pokemon.cp_multiplier <= '0.784637' && pokemon.additional_cp_multiplier > '0') {pokemon_level = '39.5';}
+	else if (pokemon.cp_multiplier <= '0.790300' && pokemon.additional_cp_multiplier <= '0') {pokemon_level = '40';}
+	else if (pokemon.cp_multiplier <= '0.790300' && pokemon.additional_cp_multiplier > '0') {pokemon_level = '40.5';}
+
```
Comment (friscoMad) [inline core/js/trainer.content.js]: Not sure but if possible a formula will be better.
Anyway there is no need for 2 checks for each level just and a second if with additional_cp_multiplier for the +0.5

**Comment by titanicboyke** [comment]
a nice suggestion but I won't use it! too much information scares visitors away

**Comment by Obihoernchen** [comment]
In your Screenshot 3419 Dragoran is L43?
I'm not sure about this change per default either.
What's about another config option? :laughing:

**Comment by pki0** [comment]
@Obihoernchen Oh yes, thanks for finding. Like always a str/float problem: 4 > 10 ;-)

- fixed

@titanicboyke I understand this, but you can easy calculate the pokemonlevel if you know IV and CP. So for me it makes no difference

**Comment by pki0** [comment]
The idea for that came from my users. They want to see their gym pokemon level and decide if they are worth to push to L40 or if it's a waste of candy.
That was my motivation @Obihoernchen

**Comment by titanicboyke** [comment]
a suggestion,
allows the user the choice by this on or off in varables.json?

**Comment by SiteEffect** [comment]
@titanicboyke another ``variables.json`` variable hurts for such a small "feature". However, I think this could be addressed in the config page we'd like to make.
Coming to the actual PR, I like the ability to see the level but it should be placed / presented smoother and not adding another line, imo.

**Comment by pki0** [comment]
I'll close this until I think about a rework.

**Pull request closed without merging**